Cargo.toml

```diff
@@ -73,3 +73,266 @@ opt-level = 3
 inherits = "release"
 lto = true
 codegen-units = 1
+
+[patch."https://github.com/paritytech/substrate"]
```

Comment:
This will be removed once I have figured out a suitable path for the wasmtime version situation.